tracer/test/test-applications/azure-functions/Samples.AzureFunctions.V4InProcess/Properties/launchSettings.json

```diff
@@ -0,0 +1,24 @@
+{
+  "profiles": {
+    "Samples.AzureFunctions.AllTriggers": {
+      "commandName": "Project",
+      "environmentVariables": {
+        "CORECLR_ENABLE_PROFILING": "1",
+        "CORECLR_PROFILER": "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
+        "CORECLR_PROFILER_PATH": "$(SolutionDir)shared\\bin\\monitoring-home\\tracer\\win-$(Platform)\\Datadog.Trace.ClrProfiler.Native.dll",
+
+        "DD_DOTNET_TRACER_HOME": "$(SolutionDir)shared\\bin\\monitoring-home\\tracer",
+        "DD_VERSION": "1.0.0",
+
+        // local.settings.json only applies to the function script, not the host, so this needs to exist here
```

Comment:
We can have comments in json? I thought it wouldn't deserialize

Reply:
It's not valid JSON, but a lot of JSON parsers ignore them because not allowing comments is one of the worst things about JSON 😉 This is one such parser (This file was copied from the `V3InProcess` sample, so it definitely works)

Reply:
:til: thank you